sbncode/CAFMaker/FillReco.cxx

```diff
@@ -155,16 +156,28 @@ namespace caf
     srmatch.flashYWidth = match.flashYWidth;
     srmatch.flashZWidth = match.flashZWidth;
     srmatch.flashClassification = static_cast<int>(match.flashClassification);
-    for(const auto& matchedCRTHit : match.matchedCRTHits){
-      std::cout << "CRTPMTTimeDiff = "<< matchedCRTHit.PMTTimeDiff << "\n";
+    srmatch.matchedCRTHits.clear();
+    caf::SRMatchedCRT matchedCRT;
+    if(!match.matchedCRTHits.empty()){
+      matchedCRT.PMTTimeDiff = crtmatch.PMTTimeDiff; 
+      matchedCRT.time = crtmatch.time;
+      matchedCRT.sys = crtmatch.sys;
+      matchedCRT.region = crtmatch.region;
+      matchedCRT.position = SRVector3D(crtmatch.position.X(), crtmatch.position.Y(), crtmatch.position.Z());
+      std::cout << "CRTPMTTimeDiff = "<< matchedCRT.PMTTimeDiff << "\n";
+    }
+    std::cout << "CRTPMTTimeDiff = "<< matchedCRT.PMTTimeDiff << "\n";
```

Comment:
Quick question: What would line 169 return if match.matchedCRTHits is empty?
Also in general I would move the std::cout to a mf::Log

Reply:
if match.matchedCRTHits is empty, the matchedCRT variables are set to their defaults defined in [StandardRecord/SRCRTPMTMatch.cxx](https://github.com/SBNSoftware/sbnanaobj/blob/develop/sbnanaobj/StandardRecord/SRCRTPMTMatch.cxx#L11), so matchedCRT.PMTTimeDiff would return `std::numeric_limits<double>::lowest()`